src/app/dataset/dataset.component.html

```diff
@@ -36,10 +36,13 @@
       </div>
       <div class="metis-dashboard-right">
         <div
-          *ngIf="publicationFitnessWarning(datasetData.publicationFitness); let warning"
-          class="unfit-to-publish"
+          *ngIf="
+            publicationFitnessWarningAndClass(datasetData.publicationFitness);
+            let warningAndClass
+          "
+          class="unfit-to-publish {{ warningAndClass.cssClass }}"
```

Comment:
Since the css class `partial-fitness` extends `unfit-to-publish` do you even need two classes here? Perhaps you can move `unfit-to-publish` to be the UNFIT alternative in the `warningAndClass` object?

Reply:
Done